### DISCUSSION TITLE
Can you provide sample code for getting object(entity) from tapped location on iOS

### DISCUSSION THREAD

**Discussion by hanjinghe** [google/filament#6489] [Q&A]
I would like to get sample code how to use pick function in filament::View.

Thank you

**Comment by bejado** [comment]
Our `gltf_viewer` sample contains picking functionality: https://github.com/google/filament/blob/298e54c32a27b58a7d2f7e7cdc4275c574c493ad/samples/gltf_viewer.cpp#L450-L458

**Reply by dandingol03** [reply]
> Our `gltf_viewer` sample contains picking functionality:
> 
> https://github.com/google/filament/blob/298e54c32a27b58a7d2f7e7cdc4275c574c493ad/samples/gltf_viewer.cpp#L450-L458

Hi, the code you reference is desktop sample. How to get the touch pos on ios device.
